### PR TITLE
storaged: Move "Format" button into Content panel

### LINKED ISSUE #1304: Storage: "Format Disk" is not discoverable.
The "Format" for the whole device (that allows people to create partition tables) is in the menu at the top of the page.  For RAIDs, it is quite hidden even and only visible when people explicitly open the action menu.

https://bugzilla.redhat.com/show_bug.cgi?id=1142251

One idea is to move it down into the header of the "Content" panel.

### PR DESCRIPTION
**Pull request by mvollmer**
And call it "Create partition table".

Fixes #1304

### CODE REVIEW

**Comment by dperpeet** [comment]
you need to change the test to reflect this change as well
```
# ----------------------------------------------------------------------
# testRaid (check_storage_mdraid.TestStorage)
#
Page error: .panel-heading:contains("RAID Device") .btn-group a:contains("Format") not found
not ok 54 testRaid (check_storage_mdraid.TestStorage)
Traceback (most recent call last):
  File "./check-storage-mdraid", line 127, in testRaid
    self.raid_action("Format")
  File "./check-storage-mdraid", line 57, in raid_action
    self.browser.click_action_btn ('.panel-heading:contains("RAID Device") .btn-group', action)
  File "/home/hubbot/hubbot/13d97747967d152557c4f1714b41c8eb72068579_r7_x86-64/test/testlib.py", line 337, in click_action_btn
    self.click(sel + ' a:contains("%s")' % entry, True);
  File "/home/hubbot/hubbot/13d97747967d152557c4f1714b41c8eb72068579_r7_x86-64/test/testlib.py", line 150, in click
    self.call_js_func('ph_click', selector, force)
  File "/home/hubbot/hubbot/13d97747967d152557c4f1714b41c8eb72068579_r7_x86-64/test/testlib.py", line 142, in call_js_func
    return self.phantom.eval("%s(%s)" % (func, ','.join(map(jsquote, args))))
  File "/home/hubbot/hubbot/13d97747967d152557c4f1714b41c8eb72068579_r7_x86-64/test/testlib.py", line 601, in <lambda>
    return lambda *args: self._invoke(name, *args)
  File "/home/hubbot/hubbot/13d97747967d152557c4f1714b41c8eb72068579_r7_x86-64/test/testlib.py", line 624, in _invoke
    raise Error(res['error'])
Error: .panel-heading:contains("RAID Device") .btn-group a:contains("Format") not found
```

**Comment by mvollmer** [comment]
> the tests need to reflect this change as well

Done.

File: pkg/storaged/index.html [hunk 1]
```diff
@@ -332,7 +332,13 @@ <h1 translatable="yes">The "storaged" API is not available on this system.</h1>
 
   <script id="content-tmpl" type="x-template/mustache">
     <div class="panel panel-default" id="content">
-      <div class="panel-heading">{{Title}}</div>
+      <div class="panel-heading">
+        <span>{{Title}}</span>
+        {{#path}}
+        <button class="btn btn-default storage-privileged" style="float:right" translatable="yes"
```
Comment (stefwalter) [inline pkg/storaged/index.html]: This should use the ```pull-right``` class.

**Comment by andreasn** [comment]
Looks good.

File: pkg/storaged/index.html (hunk 1 above)
Comment (mvollmer) [inline pkg/storaged/index.html]: Fixed

**Pull request closed without merging**